lib/api_browser/app/views/directives/attribute_description/member_options.html

```diff
@@ -0,0 +1,4 @@
+<dt>Member type values</dt>
+<dd class="well">
+	<attribute-description attribute="{options: row.value, description: 'These settings apply to the contained type:'}"></attribute-description>
```

Comment:
I know this isn't all on you, but going forward please can we use either tabs or spaces for indentation in the templates, not both.